config/assets.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# This file is located at `config/assets.rb` of your module.
+
+# Define the base path of your module. Please note that `Rails.root` may not be
+# used because we are not inside the Rails environment when this file is loaded.
+# base_path = File.expand_path("..", __dir__)
+
+# If you want to import some extra SCSS files in the Decidim main SCSS file
+# without adding any extra stylesheet inclusion tags, you can use the following
+# method to register the stylesheet import for the main application. This would
+# include an SCSS file at `app/packs/stylesheets/your_app_extensions.scss` into
+# the Decidim's main SCSS file.
+Decidim::Webpacker.register_stylesheet_import("stylesheets/coincidim")
```

Comment:
There's an alternative to this (and this file can be empty).
Require the endpoints in the default file for a decidim application, like don in here:
https://github.com/Platoniq/decidim-indices/blob/main/app/packs/stylesheets/decidim/decidim_application.scss